koku/api/provider/view.py

```diff
@@ -130,6 +130,9 @@ def get_queryset(self):
         by filtering against a `user` object in the request.
         """
         queryset = Provider.objects.none()
+        stat = self.request.query_params.get('stats', None)
+        if stat is not None and stat != 'false' and stat != 'true':
```

Comment:
I don't think this is case insensitive as it wouldn't match 'False' or 'True'. I would suggest a `stat.lowercase()`.